tasks/src/unconditional-image-generation/data.ts

```diff
@@ -0,0 +1,55 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "NFT images dataset of 'cryptopunks' for unconditional image generation.",
+			id:          "huggingnft/cryptopunks",
+		},
+		{
+			description: "NFT images dataset of 'alpacadabraz' for unconditional image generation.",
+			id:          "huggingnft/alpacadabraz",
+		},
+	],
+	demo: {
+		inputs: [
+			{
+				label:   "Input",
+				content: "",
```

Comment:
You can check out this demo for inspiration on how it should look like 🙂 https://huggingface.co/spaces/fusing/celeba-diffusion

Reply:
I think you can simply add a number instead. And as an output it's best if we don't use a face 🙂

Reply:
Thanks again! Since it is better not to use a face I am now using this model https://huggingface.co/spaces/huggan/huggingnft to generate the output and also created the inputs based on it. What do you think? Thanks again for your kind support. 🙂